package.json

```diff
@@ -32,14 +32,14 @@
   "homepage": "https://github.com/UKHomeOffice/asl-internal-ui#readme",
   "dependencies": {
     "@asl/pages": "^31.1.0",
```

Comment:
asl-pages also depends on asl-constants so update that first and add the new version here.

Reply:
@jeff-horton-ho-sas, done:
Update dependency asl-pages from 31.1.0 to 31.1.1